src/test/java/com/epam/tc/nitcenkov/hw2/ex1/FirstExerciseTest.java

```diff
@@ -0,0 +1,98 @@
+package com.epam.tc.nitcenkov.hw2.ex1;
+
+import com.epam.tc.nitcenkov.hw2.GeneralExerciseTest;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class FirstExerciseTest extends GeneralExerciseTest {
+
+    @Test
+    public void firstExerciseTest() {
+
+        //Open test site by URL
+        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+
+        //Assert Browser title
+        softAssertions.assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+
+        //Perform login
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(
+            webDriver.findElement(By.id("user-icon")))).click();
+        webDriver.switchTo().activeElement();
+        webDriverWait.until(ExpectedConditions
+            .visibilityOf(webDriver.findElement(By.id("name")))).sendKeys("Roman");
+        webDriverWait.until(ExpectedConditions
+            .visibilityOf(webDriver.findElement(By.id("password")))).sendKeys("Jdi1234");
+        webDriverWait.until(ExpectedConditions
+            .visibilityOf(webDriver.findElement(By.id("login-button")))).click();
+
+        //Assert Username is logged
+        String actualUserName =
+            webDriverWait.until(ExpectedConditions.visibilityOf(webDriver.findElement(By.id("user-name")))).getText();
+        softAssertions.assertThat(actualUserName).isEqualTo("ROMAN IOVLEV");
+
+        //Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<String> expectedMenuButtons = Arrays.asList("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
+        List<WebElement> menuButtons = webDriverWait.until(ExpectedConditions.visibilityOfAllElements(
+            webDriver.findElements(By.cssSelector("ul.nav > li"))));
+        softAssertions.assertThat(menuButtons.size()).isEqualTo(4);
+        for (WebElement element : menuButtons
+        ) {
+            softAssertions.assertThat(element.isDisplayed()).isTrue();
+        }
```

Comment:
Please use something like `softAssertions.assertThat(...).allMatch(WebElement::isDisplayed)`
Because in general assertions inside loop is a bad practice